data/data_download/wdpa/Makefile

```diff
@@ -34,31 +35,47 @@ merge-simplify: data/wdpa/simp/0 data/wdpa/simp/1 data/wdpa/simp/2
 
 # @debt
 # need to review island removal
-data/wdpa/simp/%: data/wdpa/geojson/%/
+data/wdpa/simp/%: data/wdpa/geojson/%
 	mkdir -p $@
-	mapshaper-xl 16gb -i $<*.json snap \
-				-simplify 10% planar keep-shapes \
-				-filter-islands min-vertices=3 remove-empty \
-				-filter-slivers \
+	mapshaper-xl 16gb -i $</*.json snap \
+				-simplify 15% planar keep-shapes \
+				-filter-islands min-vertices=3 min-area=100 remove-empty \
+				-filter-slivers min-area=100 \
 				-clean rewind \
 				-o $@ format=geojson force
+	rm -rf $<
 
 # @debt
 # need to review a better way to split the files than using the iucn category
-data/wdpa/geojson/%/: data/wdpa/WDPA_WDOECM_wdpa_shp%/
-	mkdir -p $@
-	mapshaper-xl 16gb -i $<*-polygons.shp name='$(shell basename $<)' \
-					-split IUCN_CAT \
-					-o $@ format=geojson precision=0.0000001
-
-data/wdpa/WDPA_WDOECM_wdpa_shp0/ data/wdpa/WDPA_WDOECM_wdpa_shp1/ data/wdpa/WDPA_WDOECM_wdpa_shp2/: data/wdpa/WDPA_WDOECM_wdpa_shp.zip
-	unzip -u $< -d data/wdpa/out
+data/wdpa/geojson/%: data/wdpa/data
+	@for f in data/wdpa/data/*.shp; \
+		do filename=`(basename -s .shp "$$f")`; \
+		readarray -td '_' a < <(echo "$${filename}_"); unset 'a[-1]'; \
+		declare -p a; \
+		echo "$${a[-1]}"; \
+		mkdir -p $@; \
+		mapshaper-xl 16gb -i $$f name="$$filename" \
+						-split IUCN_CAT \
+						-o $@ format=geojson precision=0.0000001; \
+	done
+	rm -rf $<
 
-	unzip data/wdpa/out/$(shell basename $@).zip \
-			-d data/wdpa/"$(shell basename $@)"
+data/wdpa/data: data/wdpa/wdpa.zip
+	@unzip -u $< -d data/wdpa/out
+	@for f in data/wdpa/out/*.zip; \
+		do echo "$$(basename $$f)"; \
+		mkdir -p $@/temp; \
+		unzip "$$f" -d $@/temp; \
+		mv $@/temp/*-polygons.shp $@/$$(basename $$f .zip).shp; \
+   		mv $@/temp/*-polygons.shx $@/$$(basename $$f .zip).shx; \
+		mv $@/temp/*-polygons.dbf $@/$$(basename $$f .zip).dbf; \
+      	mv $@/temp/*-polygons.prj $@/$$(basename $$f .zip).prj; \
+        mv $@/temp/*-polygons.cpg $@/$$(basename $$f .zip).cpg; \
```

Comment:
I think this may potentially break if at some point upstream starts putting in a folder more than one file with suffix `-polygons.shp` and so on, but at that stage the pipeline will break and we'll fix it.
It may be slightly worse in case they split some of these files into two or more with different prefix only for some areas, because the error would be flaky, but I'd say that this is unlikely in general, so I can take my tin foil hat off here :)

Reply:
hopefully, when this breaks we will no longer need it :crossed_fingers: . Let me review the issues in the log file.

Reply:
after reviewing and reruning the seed on the instance... things are working as expected, you can check that the data is populated in the db